Alignment/TrackerAlignment/test/cosmicRateAnalyzer_cfg.py

```diff
@@ -0,0 +1,39 @@
+import FWCore.ParameterSet.Config as cms
+
+import os
+
+process = cms.Process("Demo")
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+process.load('Configuration.StandardSequences.MagneticField_cff') # B-field map
+process.load('Configuration.Geometry.GeometryRecoDB_cff') # Ideal geometry and interface
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff") # Global tag
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag.globaltag = GlobalTag("80X_dataRun2_Prompt_v2")
+
+process.options = cms.untracked.PSet(
+     SkipEvent= cms.untracked.vstring("ProductNotFound"), # make this exception fatal
+  )
+process.maxEvents=cms.untracked.PSet(input=cms.untracked.int32(50000)) 
+
+import FWCore.Utilities.FileUtils as FileUtils
+ 
+readFiles = cms.untracked.vstring()
+
+
+readFiles = cms.untracked.vstring( FileUtils.loadListFromFile (os.environ['CMSSW_BASE']+'/src/Alignment/TrackerAlignment/test/'+'fileList.txt') )
+process.source = cms.Source("PoolSource",
+			   fileNames = readFiles,
+			   )
+
+process.TFileService = cms.Service("TFileService", fileName = cms.string("Cosmic_rate_tuple.root") )
+process.demo = cms.EDAnalyzer("CosmicRateAnalyzer",
```

Comment:
@skeshri when you have added the filldescriptions method you can load the auto-generated file here and modify only the parameters you want